### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by superzuc**
I've made another new project. I think I've figured it out where was the problem. Thanks

https://superzuc.github.io/layout_html-form/

https://superzuc.github.io/layout_html-form/report/html_report/

### CODE REVIEW

**Review by qoa11a — CHANGES_REQUESTED** [review]
image
let's work on passing all tests. If you need help with it ask for help in chat

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,207 @@
...
+        <legend>Personal information</legend>
+
+        <div>
+          <label class="form_field" for="sname">Surname:</label>
+          <input
+            class="form_element"
+            type="text"
+            name="surname"
+            id="sname"
+          ><br>
```
Comment (qoa11a) [inline src/index.html]:
```suggestion
          <input
            class="form_element"
            type="text"
            name="surname"
            id="sname"
          >
          <br>
```
fix it everywhere

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,207 @@
...
+          <label class="form_field form_element" for="cats">Do you love cats?</label>
+          <label for="yes">
+            <input
+              class="form_element"
+              type="radio"
+              name="cats"
+              id="yes"
+              checked
+            >
+          Yes
```
Comment (qoa11a) [inline src/index.html]:
```suggestion
            <input
              class="form_element"
              type="radio"
              name="cats"
              id="yes"
              checked
            >
            Yes
```

**Review by qoa11a — CHANGES_REQUESTED** [review]
good job! 👍 
few small fixes

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,215 @@
...
+        <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="6"
+          maxlength="20"
+          required
+        >
+
+      </fieldset>
```
Comment (polosanya) [inline src/index.html:111]:
```suggestion
        >
      </fieldset>
```
Don't leave newlines between parent and child tags

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,215 @@
...
+            <input
+              class="form_element"
+              type="radio"
+              name="cats"
+              id="no"
+            >
+            No
+          </label>
+
+        </div>
```
Comment (polosanya) [inline src/index.html:138]:
```suggestion
          </label>
        </div>
```
Fix all cases

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,215 @@
...
+        <legend>Additional info</legend>
+
+        <div>
+          <label class="form_field" for="comments">Comments</label>
+          <textarea
+            class="form_element"
+            name="comments"
+            id="comments"
+            autocomplete="off">
+          </textarea>
```
Comment (polosanya) [inline src/index.html:200]:
```suggestion
            autocomplete="off"
          ></textarea>
```
image

image

**Review by polosanya — CHANGES_REQUESTED** [review]
Let's polish it up)

**Pull request closed without merging**